var/spack/repos/jcsda-emc/packages/mapl/package.py

```diff
@@ -32,18 +32,15 @@ class Mapl(CMakePackage):
     version('2.7.2', sha256='8f123352c665c434a18ff87304a71a61fb3342919adcccfea2a40729992d9f93')
     version('2.7.1', sha256='8239fdbebd2caa47a232c24927f7a91196704e35c8b7909e1bbbefccf0647ea6')
 
+    # Versions later than 3.14 remove FindESMF.cmake
+    # from ESMA_CMake.
     resource(
         name='esma_cmake',
         git='https://github.com/GEOS-ESM/ESMA_cmake.git',
-        tag='v3.4.3')
-
-    resource(
-        name='CMakeModules',
-        git='https://github.com/NOAA-EMC/CMakeModules.git',
-        tag='v1.2.0')
+        tag='v3.13.0')
 
     # Patch to configure Apple M1 chip in x86_64 Rosetta 2 emulator mode
-    patch('esma_cmake_apple_m1.patch', when='@:2.8.1')
+    patch('esma_cmake_apple_m1_rosetta.patch')
```

Comment:
Can this be applied to versions earlier than 2.8.1?

Reply:
The patch is for ESMA_CMake which is downloaded as a resource.

Reply:
But good question. Let me try some older versions.